kubevirt-disk-uploader.yaml

```diff
@@ -54,21 +54,13 @@ spec:
             secretKeyRef:
               name: kubevirt-disk-uploader-credentials
               key: registryHostname
-      command: ["/usr/bin/kubevirt-disk-uploader.sh"]
+      command: ["/usr/bin/run-uploader.sh"]
       args: ["example-vm", "example-vm-exported:latest", "disk.img.gz"]
       resources:
         requests:
-          memory: 512Mi
-        limits:
           memory: 1Gi
-      securityContext:
-        capabilities:
-          add:
-            # Building using buildah requires a properly installed shadow-utils package (which in turn requires SETFCAP)
-            # https://www.redhat.com/sysadmin/podman-inside-kubernetes
-            - "SETFCAP"
-        # Buildah inside a pod requires root privileges
-        privileged: true
+        limits:
+          memory: 10Gi
```

Comment:
This is a lot, do we have to go so high?

Reply:
If you use cache=none it should never use over 600M. This is because cache=none bypasses the page cache and writes directly to storage device. Which image were you using that brought you to over 2gb?

Reply:
Thanks, I'll keep it in a backlog and will take care of it later.